src/app/components/main/Main.types.ts

```diff
@@ -0,0 +1,4 @@
+export type MainProps = {
+  screenIsMobile: boolean;
+  handleMobileFiltersToggle: () => void;
```

Comment:
To jest callback, więc powinien być nazwany jak callback z perspektywy komponentu, w którym jest, czyli `onFilterClick`